fili-wikipedia-example/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM druidio/example-cluster:latest
+LABEL maintainer="https://groups.google.com/forum/#!forum/fili-developers"
+
+# fili github information
+ENV GITHUB_OWNER=yahoo BRANCH=master
```

Comment:
`BRANCH=master` goes to the next line (Fili code style favors multiline declaration)